core/src/main/java/com/adobe/granite/translation/connector/bootstrap/core/impl/BootstrapTranslationServiceImpl.java

```diff
@@ -282,60 +319,121 @@ public void addTranslationJobComment(String strTranslationJobID, Comment comment
 
     @Override
     public InputStream getTranslatedObject(String strTranslationJobID, TranslationObject translationObj)
-        throws TranslationException {
-        log.trace("BootstrapTranslationServiceImpl.getTranslatedObject");
-        return bootstrapTmsService.getTmsObjectTranslatedInputStream(strTranslationJobID, getObjectPath(translationObj));
+      throws TranslationException {
+      log.trace("BootstrapTranslationServiceImpl.getTranslatedObject");
+      String labels = String.format("%s,status=TRANSLATED", strTranslationJobID);
+      String objectPath = String.format("%s.%s", getObjectPath(translationObj), exportFormat);
+      try {
+        SourceFile[] files = liltApiClient.getFiles(labels);
+        // loop backwards through the list since the most recent files will be at the end.
+        for (int i = files.length - 1; i >= 0; i--) {
+          SourceFile file = files[i];
+          if (Objects.equals(file.name, objectPath)) {
+            log.warn("Downloading the translated lilt file {}", file.id);
+            String translation = liltApiClient.downloadFile(file.id);
+            return new ByteArrayInputStream(translation.getBytes());
+          }
+        }
+      } catch (Exception e) {
+        log.warn("error during getTranslatedObject {}", e);
```

Comment:
Log4j has a special method signature when there is only one object parameter that is an exception/throwable -- in this case, it will automatically log the full stack track.  In this case, you don't want to interpolate it with `{}`, which will either only log the error message of the exception, or maybe just log a raw `{}`.  (In this specific case, I'm not sure.)

So better to use:
```suggestion
        log.warn("error during getTranslatedObject", e);
```

There are a bunch of instances like this, I didn't flag them all.